hermes/auth_backends.py

```diff
@@ -20,6 +20,24 @@ class NotInKafkaUsers(PermissionDenied):
     pass
 
 
+class HermesTokenAuthentication(TokenAuthentication):
+    def authenticate(self, request, **kwargs):
+        """Override this method to verify hop credential is valid within profile and regenerate it if not
+
+        Notes:
+         * the user is a django.contrib.auth.User instance
+         * the safe way to the username is user.get_username()
+
+        Extends base class method.
+        """
+        logger.warning("Trying to authenticate!!!")
+        auth = super().authenticate(request, **kwargs) # django.contrib.auth.models.User
```

Comment:
It might be kind to our six-months-from-now selves to add some type-hints and/or indicate what `auth` is, how is it truthy? and what `auth[0]` is.